staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go

```diff
@@ -51,3 +55,36 @@ func (oa OpenAPI) Install(c *restful.Container, mux *mux.PathRecorderMux) (*hand
 
 	return openAPIVersionedService, spec
 }
+
+func (oa OpenAPI) InstallV3(c *restful.Container, mux *mux.PathRecorderMux) *handler3.OpenAPIService {
+	w := c.RegisteredWebServices()
```

Comment:
any need to have that that far up? Just use `range c.RegisteredWebServices()` below.